[2025-02-21T08:25:20.473+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:25:18.976115+00:00 [queued]>
[2025-02-21T08:25:20.490+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:25:18.976115+00:00 [queued]>
[2025-02-21T08:25:20.490+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T08:25:20.509+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): consume_kafka_messages> on 2025-02-21 08:25:18.976115+00:00
[2025-02-21T08:25:20.520+0000] {standard_task_runner.py:57} INFO - Started process 492 to run task
[2025-02-21T08:25:20.524+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_consumer_dag', 'consume_kafka_messages', 'manual__2025-02-21T08:25:18.976115+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/consume.py', '--cfg-path', '/tmp/tmplblimvnn']
[2025-02-21T08:25:20.525+0000] {standard_task_runner.py:85} INFO - Job 29: Subtask consume_kafka_messages
[2025-02-21T08:25:20.594+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:25:18.976115+00:00 [running]> on host c0c1f1afe7d6
[2025-02-21T08:25:20.709+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_consumer_dag' AIRFLOW_CTX_TASK_ID='consume_kafka_messages' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T08:25:18.976115+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T08:25:18.976115+00:00'
[2025-02-21T08:25:20.714+0000] {logging_mixin.py:154} INFO - Consuming messages from topic 'from_mysql_transaction'...
[2025-02-21T08:25:21.714+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:22.715+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:23.716+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:24.717+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:25.717+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:26.718+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:27.719+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:28.719+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:29.720+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:30.721+0000] {logging_mixin.py:154} INFO - 1
[2025-02-21T08:25:30.722+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T08:25:30.736+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_consumer_dag, task_id=consume_kafka_messages, execution_date=20250221T082518, start_date=20250221T082520, end_date=20250221T082530
[2025-02-21T08:25:30.789+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T08:25:30.813+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
